deployed-contracts/xvs-multichain.md

```diff
@@ -38,4 +44,10 @@
 
 * XVSBridgeAdmin: [`0x19252AFD0B2F539C400aEab7d460CBFbf74c17ff`](https://testnet.opbnbscan.com/address/0x19252AFD0B2F539C400aEab7d460CBFbf74c17ff)
 * XVSProxyOFTDest: [`0xA03205bC635A772E533E7BE36b5701E331a70ea3`](https://testnet.opbnbscan.com/address/0xA03205bC635A772E533E7BE36b5701E331a70ea3)
-* XVS: [`0xc2931B1fEa69b6D6dA65a50363A8D75d285e4da9`](https://testnet.opbnbscan.com/address/0xc2931B1fEa69b6D6dA65a50363A8D75d285e4da9)
\ No newline at end of file
+* XVS: [`0xc2931B1fEa69b6D6dA65a50363A8D75d285e4da9`](https://testnet.opbnbscan.com/address/0xc2931B1fEa69b6D6dA65a50363A8D75d285e4da9)
+
+### Arbitrum Sepolia
+
+* XVSBridgeAdmin: [`0xc94578cacc89a29b044a0a1d54d20d48a645e5c8`](https://sepolia.arbiscan.io/address/0xc94578cacc89a29b044a0a1d54d20d48a645e5c8)
```

Comment:
```suggestion
* XVSBridgeAdmin: [`0xc94578caCC89a29B044a0a1D54d20d48A645E5C8`](https://sepolia.arbiscan.io/address/0xc94578caCC89a29B044a0a1D54d20d48A645E5C8)
```

Reply:
Both addresses are same it is just the uppercase and lowercase thing which will not affect. So, we can just ignore it.

Reply:
I suggested the change because we are using the uppercase and lowercase format in the rest of the addresses